cartoframes/client/client.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABCMeta, abstractmethod
+
+
+class ClientBase():
```

Comment:
I would name this client as `BaseClient`. Also the file `base_client.py`

Reply:
Right!